modules/_labs/menu/react/lib/Menu.tsx

```diff
@@ -89,16 +90,22 @@ export default class Menu extends React.Component<MenuProps, MenuState> {
     if (this.props.isOpen && !prevProps.isOpen) {
       this.setInitialSelectedItem();
     }
-    if (this.props.isOpen && this.menuRef.current) {
-      this.menuRef.current.focus();
-    }
+    this.animateId = requestAnimationFrame(() => {
```

Comment:
We delay the focus checking, but track the `requestAnimationId` so we can cancel on unmount just in case the component is mounted before this callback triggers.